server/utils.py

```diff
@@ -67,6 +63,22 @@ def server_time_obj(time, course):
     # Store using UTC on the server side.
     return time.astimezone(pytz.utc)
 
+def next_week_obj(course):
+    """Get a datetime object representing 1 week from now at 11:59:59 pm."""
+    next_week_date = dt.datetime.now(course.timezone) + dt.timedelta(weeks=1)
+    midnight_time = dt.time(hour=23, minute=59, second=59, microsecond=0,
```

Comment:
I think there's some danger in using pytz time zones with date time. I'll look into it into when I get back on my computer